.circleci/config.yml

```diff
@@ -0,0 +1,46 @@
+version: 2.1
+orbs:
+  ruby: circleci/ruby@1.8.0
+
+jobs:
+  build:
+    docker:
+      - image: "cimg/openjdk:17.0.3"
+    steps:
+      - checkout
+      - run: gradle build
+  unitTests:
+    docker:
+      - image: "cimg/openjdk:17.0.3"
+    steps:
+      - checkout
+      - run: gradle test
+  acceptanceTests:
+    docker:
+      - image: "cimg/openjdk:17.0.3"
+    steps:
+      - checkout
+      - run:
+          command: gradle run
+          background: true
+      - run: git submodule update --init --recursive
+      - ruby/install:
+          version: '2.6.8'
```

Comment:
During our pairing, I think we found out that the ruby version `2.5.1` specified for `http_server_spec` is out of date and certain gem requires a higher version in order to work. Do you want to update the `http_server_spec` to use version `2.6.8` instead?

Reply:
Pairing with Eli at 11 EST to fix this and some commits I made on the repo by mistake 😶